core/src/main/java/org/infinispan/interceptors/impl/CacheWriterInterceptor.java

```diff
@@ -74,7 +72,7 @@
    private final boolean trace = getLog().isTraceEnabled();
    PersistenceConfiguration loaderConfig = null;
    final AtomicLong cacheStores = new AtomicLong(0);
-   protected PersistenceManager persistenceManager;
+   protected TransactionalPersistenceManager persistenceManager;
```

Comment:
Yes I will change it back. I went with Transactional, as it meant that the method call would be more explicit, i.e. ```writeToAllNonTxStores``` but I think ```writeToAllStores``` through the ```PersistenceManager``` interface should be sufficient.